content/docs/_index.md

```diff
@@ -6,39 +6,37 @@ weight: 1
 
 ## About Falco
 
-Falco is a behavioral activity monitor designed to detect anomalous activity in your applications. Powered by sysdig’s system call capture infrastructure, Falco lets you continuously monitor and detect container, application, host, and network activity... all in one place, from one source of data, with one set of rules.
+Falco is a behavioral activity monitor designed to detect anomalous activity in your applications. Powered by [Sysdig](https://sysdig.com/)'s [system call capture](https://sysdig.com/blog/fascinating-world-linux-system-calls/) infrastructure, Falco lets you continuously monitor and detect container, application, host, and network activity, all in one place, from one source of data, with one set of [rules](rules).
```

Comment:
Just wondering whether is better to link https://github.com/draios/sysdig rather than the website and/or blog. WDYT Luc?

Reply:
I'll switch the sysdig link to the GiHub repo but I think linking to an informative blog post is totally fine